docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -56,6 +56,18 @@ The following table lists the attributes you can pass to the `create` command:
 | _[]_
 | The list of extensions to add to the project (comma-separated)
 
+| `platformGroupId`
+| `io.quarkus`
+| The group id of the platform you want to use.
+
+| `platformArtifactId`
+| `quarkus-bom`
+| The artifact id of the platform you want to use.
+
+| `platformVersion`
+| `quarkus-bom`
```

Comment:
This is incorrect. It should be a version (or probably more something saying how the version is resolved).